lib/app.js

```diff
@@ -108,6 +108,10 @@ const MAPPINGS = {
   genreId: ['ds:5', 1, 2, 79, 0, 0, 2],
```

Comment:
`genre` returns the default Category as far as I can tell. As for `familyGenre` - don't know what it is, I haven't seen it once. 
`genre` overlaps with my `categories` (as the first item) most of the time, but not always.

Reply:
I can confirm that family genre is now empty, even in apps in the FAMILY category. Can you remove those two fields as part of this PR?

Reply:
Do you mean `familyGenre` and `familyGenreId` by "two fields" or `familyGenre` and `genre` (with their Ids)? 
I'll do it, but it can break someone's code, probably. The version update is on you, I'm weak in those.

Reply:
I meant the two family genre fields